packages/airnode-deployer/src/infrastructure/aws.test.ts

```diff
@@ -60,44 +59,38 @@ const cloudProvider = {
 const bucketName = 'airnode-aabbccdd0011';
 const fileContent = 'file content';
 const filePath = '/path/to/config.json';
-const bucketFilePath = '0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace/dev/1662559204/config.json';
+const bucketFilePath = '0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace/dev/1662559204554/config.json';
```

Comment:
Mhh, is this needed? :D

Reply:
What? The change from seconds to milliseconds? Not for these tests but I'm using the same mock dicrectory structure everywhere and it is needed for the new tests. Also, I want to be consistent with the reality, it should have been in milliseconds from the beginning.